dlp/google/cloud/proto/privacy/dlp/v2beta1/dlp_pb2_grpc.py

```diff
@@ -62,23 +62,24 @@ class DlpServiceServicer(object):
   """
 
   def InspectContent(self, request, context):
-    """Find potentially sensitive info in a list of strings.
+    """Finds potentially sensitive info in a list of strings.
```

Comment:
These managed to get worse. Was this just an upstream change to the comments that created this docstring?